pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -179,3 +182,10 @@ type StorageClusterList struct {
 func init() {
 	SchemeBuilder.Register(&StorageCluster{}, &StorageClusterList{})
 }
+
+// ExternalResources containes a list of External Cluster Resources
+type ExternalResource struct {
+	Kind string            `json:"kind"`
+	Data map[string]string `json:"data"`
+	Name string            `json:"name"`
+}
```

Comment:
Since this is used internally and by the initialization reconciler, move this to initialization_reconciler.go.
This should not be here as this file indicates API changes.

Reply:
Yes, we were thinking of doing the same.